engineapi/instance_manager.go

```diff
@@ -474,13 +475,18 @@ func (c *InstanceManagerClient) ReplicaInstanceCreate(req *ReplicaInstanceCreate
 		return parseProcess(imapi.RPCToProcess(process)), nil
 	}
 
+	portCount := DefaultReplicaPortCount
+	if req.Replica.Spec.BackendStoreDriver == longhorn.BackendStoreDriverTypeSPDK {
+		portCount = spdktypes.DefaultReplicaReservedPortCount
+	}
```

Comment:
For SPDK replica, the current reserved port count of each replica is 5: 
https://github.com/longhorn/longhorn-spdk-engine/blob/d326caa4884afed4930bde2108b3938b4f0ddd83/pkg/types/types.go#L16

Reply:
Yes
``
portCount = spdktypes.DefaultReplicaReservedPortCount
```

Reply:
Oh, sorry. You are using different default value here.